addons/gforces/functions/fnc_pfhUpdateGForces.sqf

```diff
@@ -86,11 +80,12 @@ if (_suitCoef == 0) then {_suitCoef = 0.001};
 
 _gBlackOut = MAXVIRTUALG / _classCoef + MAXVIRTUALG / _suitCoef - MAXVIRTUALG;
 _gRedOut = MINVIRTUALG / _classCoef;
-
-["GForces", [], {format ["_g _gBO _coef: %1, %2, %3", _average, _gBlackOut, 2 * ((1.0 - ((_average - 0.30 * _gBlackOut) / (0.70 * _gBlackOut)) ^ 2) max 0) ]}] call EFUNC(common,log);
+#ifdef DEBUG_MODE_FULL
+    ["GForces", [], {format ["_g _gBO _coef: %1, %2, %3", _average, _gBlackOut, 2 * ((1.0 - ((_average - 0.30 * _gBlackOut) / (0.70 * _gBlackOut)) ^ 2) max 0) ]}] call EFUNC(common,log);
+#endif
```

Comment:
Can `TRACE_1` be used here? Would need a bit of a refactor to the debug code probably.